src/client/src/components/pages/dashboard/style.css

```diff
@@ -18,8 +18,8 @@ a{
 }
 .charts{
     display: grid;
-    grid-template-columns: 65% 30%;
-    grid-column-gap: 5%;
+    grid-template-columns: 50% 50%;
+    /* grid-column-gap: 5%; */
```

Comment:
Maybe you should delete this commented line here

Reply:
Done :+1: